docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -26,37 +26,37 @@ To start working with discounts, go to **Merchandising** > **Discount**.
+    6. Specify if the discount is exclusive. For reference information, in the **Create new discount** reference, see [General information tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#general-information-tab).
     7. Specify the validity interval (lifetime) of the discount.
-* Click **Next** or select the *Discount calculation* tab to proceed.
+* Click **Next** or select the **Discount calculation** tab to proceed.
 
-* On the *Create Discount* page, in the *Discount calculation* tab, do the following:
-    1.  In the *CALCULATOR TYPE* drop-down, select either *Percentage* or *Fixed amount*. For reference information, in the *Create new discount* reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+* On the **Discount calculation** tab, do the following:
+    1.  In the **CALCULATOR TYPE** drop-down, select either **Percentage** or **Fixed amount**. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
 
         {% info_block warningBox "Note" %}
 
         The next step varies based on the selected calculator type:
 
           a. *Fixed amount*: Enter the amounts (per currency and price mode, if applicable) to be discounted.
-          
+
           b. *Percentage*: Enter the percent value to be discounted.
 
         {% endinfo_block %}
 
-    2. Select the *DISCOUNT APPLICATION TYPE* and define which products are eligible for the discount. For reference information, in the *Create new discount* reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
+    2. Select the **DISCOUNT APPLICATION TYPE** and define which products are eligible for the discount. For reference information, in the **Create new discount** reference, see the [Discount calculation tab](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#discount-calculation-tab) section.
  * Click **Next**, or select the **Conditions** tab to proceed.
- * On the *Create Discount* page, in the *Conditions* tab, do the following:
-    1. Select the *APPLY WHEN* conditions or click **Plain query** and enter the query manually. For reference information, in the Creating a voucher guide, see the [Conditions](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-vouchers.html#conditions) section.
-    2. Enter the value for *THE DISCOUNT CAN BE APPLIED IF THE QUERY APPLIES FOR AT LEAST X ITEM(S).* field.
-* Click **Save** to create the new discount.
+ * On the **Conditions** tab, do the following:
```

Comment:
```suggestion
 * In the **Conditions** tab, do the following:
```

Reply:
According to https://developers.google.com/style/ui-elements, the _on_ preposition should be used here.